src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/CourseActivityPayRate.kt

```diff
@@ -20,6 +20,17 @@ annotation class CourseActivityPayRateDslMarker
 @NomisDataDslMarker
 interface CourseActivityPayRateDsl
 
+interface CourseActivityPayRateDslApi {
+  @CourseActivityPayRateDslMarker
+  fun payRate(
+    iepLevelCode: String = "STD",
+    payBandCode: String = "5",
+    startDate: String = "2022-10-31",
+    endDate: String? = null,
+    halfDayRate: Double = 3.2,
+  ): CourseActivityPayRate
+}
```

Comment:
Declare the abstract method in the same class that contains the builders and annotations so everything is together.